src/storage/kv/mod.rs

```diff
@@ -123,6 +123,24 @@ pub trait Snapshot: Send + Clone {
     fn upper_bound(&self) -> Option<&[u8]> {
         None
     }
+
+    /// Retrieves a version that represents the modification status of the underlying data.
+    /// Version should be changed when underlying data is changed.
+    ///
+    /// If the engine does not support data version, then `None` is returned.
+    #[inline]
+    fn get_data_version(&self) -> Option<u64> {
+        None
+    }
+
+    /// Checks whether underlying data is unchanged according to a previously retrieved version.
+    /// If data is unchanged, returns `true`.
+    ///
+    /// If the engine does not support data version, `false` is always returned.
+    #[inline]
+    fn is_data_version_matches(&self, _version: u64) -> bool {
+        false
+    }
```

Comment:
What's the purpose of this method? According to the description of `get_data_version`, this method can be implemented by `get_data_version`.